integrations/discord/websockets/websocket.go

```diff
@@ -0,0 +1,103 @@
+package websockets
+
+import (
+	"context"
+
+	"github.com/bwmarrin/discordgo"
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/integrations/internal/extrazap"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type handler struct {
+	logger        *zap.Logger
+	vars          sdkservices.Vars
+	dispatcher    sdkservices.Dispatcher
+	integration   sdktypes.Integration
+	integrationID sdktypes.IntegrationID
+}
+
+func NewHandler(l *zap.Logger, v sdkservices.Vars, d sdkservices.Dispatcher, i sdktypes.Integration) handler {
+	return handler{
+		logger:        l,
+		vars:          v,
+		dispatcher:    d,
+		integration:   i,
```

Comment:
See my other comment about moving code from `websockets` to `discord` - which would make the `integration` field unnecessary

Reply:
Done